web_m2x_options/models/ir_config_parameter.py

```diff
@@ -12,5 +12,9 @@ def get_web_m2x_options(self):
             "web_m2x_options.limit",
             "web_m2x_options.search_more",
             "web_m2x_options.m2o_dialog",
+            "web_m2x_options.field_limit_entries",
         ]
-        return self.sudo().search_read([["key", "in", opts]], ["key", "value"])
+        return {
+            res["key"]: res["value"]
+            for res in self.sudo().search_read([["key", "in", opts]], ["key", "value"])
+        }
```

Comment:
Just a non-blocking observation.
```suggestion
        values = self.sudo().search_read([["key", "in", opts]], ["key", "value"])
        return { res["key"]: res["value"] for res in values }
```